modules/qna/src/views/full/Editor/index.tsx

```diff
@@ -205,7 +208,21 @@ export default class Editor extends Component<Props> {
     return this.state.item.questions[this.props.contentLang] || []
   }
 
-  createAnswer = answer => {
+  createAnswer = async answer => {
+    const textAnswers = await Promise.map(this.itemAnswers, (a: string) =>
+      a.startsWith('#!')
+        ? this.props.bp.axios
+            .get(`content/element/${a.replace('#!', '')}`)
+            .then(({ data }) => data.formData[`text$${this.props.contentLang}`])
+        : a
+    )
+
+    if ([...textAnswers, ...this.itemAnswers].includes(answer)) {
```

Comment:
Hmm, that's over-complicated. If you have 10 content elements, it will do 10 backend calls to return the text property. It doesn't handle carousel, images, etc... so not very useful.

I suggest simply checking if any item appear more than once in the list. Be it content element Id, or simply text. The only problem is that react complains that there's two items with the same key. It's not a bug and will not impact the user.